src/app/pages/applications/dialogs/chart-upgrade/chart-upgrade-dialog.component.scss

```diff
@@ -0,0 +1,96 @@
+:host {
+  font-size: 14px !important;
+}
+
+::ng-deep .mat-dialog-content {
```

Comment:
This will affect any `mat-dialog-content` in the system.
Can we have some way of narrowing this down, i.e. by having some class on elements?